### PR TITLE
doc: Matter Bridge documentation enhancement

### PR DESCRIPTION
**Pull request by kg-nordicsemi**
- Support for Thread protocol and nRF54H20 is experimental
- There is to many unnecesary buildings commands in smart-plug section

### CODE REVIEW

**Review by maciejbaczmanski — APPROVED** [review]

File: applications/matter_bridge/doc/matter_bridge_description.rst [hunk 1]
```diff
@@ -733,21 +733,12 @@ See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information
 Configure the functionality of the Matter-Bridge device
 -------------------------------------------------------
 
-To enable the Matter smart plug functionality, run the following command:
+To enable the Matter smart plugin functionality, for example with :ref:`zephyr:nrf54h20dk_nrf54h20`, run the following command:
```
Comment (kkasperczyk-no) [inline applications/matter_bridge/doc/matter_bridge_description.rst:736]: How about using some simpler example? It's only about presenting the part required for smart plug, so -Dmatter_bridge_SNIPPET=onoff_plug, the rest is not needed. I would suggest the following: 
```suggestion
To enable the Matter smart plugin functionality, run the following command with *board_target* replaced with the board target name:
```

File: applications/matter_bridge/doc/matter_bridge_description.rst [hunk 2]
```diff
@@ -733,21 +733,12 @@ See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information
...
-         west build -b nrf54h20dk/nrf54h20/cpuapp -p -- -DSB_CONFIG_WIFI_NRF700X=y -DCONFIG_CHIP_WIFI=y -Dmatter_bridge_SHIELD=nrf700x_nrf54h20dk -DCONFIG_BRIDGED_DEVICE_BT=y -Dmatter_bridge_SNIPPET=onoff_plug
-
-   .. group-tab:: nRF70 DKs
-
-      .. code-block:: console
+.. parsed-literal::
+   :class: highlight
 
-         west build -b nrf7002dk/nrf5340/cpuapp -p -- -DCONFIG_BRIDGED_DEVICE_BT=y -Dmatter_bridge_SNIPPET=onoff_plug
+   west build -b nrf54h20dk/nrf54h20/cpuapp -p -- -DSB_CONFIG_WIFI_NRF700X=y -DCONFIG_CHIP_WIFI=y -Dmatter_bridge_SHIELD=nrf700x_nrf54h20dk -DCONFIG_BRIDGED_DEVICE_BT=y -Dmatter_bridge_SNIPPET=onoff_plug
```
Comment (kkasperczyk-no) [inline applications/matter_bridge/doc/matter_bridge_description.rst:741]: And with regards to the above comment:
```suggestion
   west build -b *board_target* -p -- -Dmatter_bridge_SNIPPET=onoff_plug
```

**Review by kkasperczyk-no — APPROVED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -225,9 +225,9 @@ Matter Bridge
 
   * The :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_ZAP_FILES_PATH` Kconfig option, which specifies ZAP files location for the application.
     By default, the option points to the :file:`src/default_zap` directory and can be changed to any path relative to application's location that contains the ZAP file and :file:`zap-generated` directory.
-  * Support for the :ref:`zephyr:nrf54h20dk_nrf54h20`.
+  * Experimental Support for the :ref:`zephyr:nrf54h20dk_nrf54h20`.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:228]:
```suggestion
  * Experimental support for the :ref:`zephyr:nrf54h20dk_nrf54h20`.
```

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -225,9 +225,9 @@ Matter Bridge
...
   * Optional smart plug device functionality.
-  * Support for the Thread protocol.
+  * Experimental Support for the Thread protocol.
```
Comment (wiba-nordic) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:230]:
```suggestion
  * Experimental support for the Thread protocol.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by wiba-nordic — APPROVED** [review]

File: applications/matter_bridge/doc/matter_bridge_description.rst (hunk 2 above)
Comment (kkasperczyk-no) [reply applications/matter_bridge/doc/matter_bridge_description.rst]: @kg-nordicsemi why haven't you applied this comment and resolved conversation? I've just noticed that the documentation was not fixed while reviewing the guide before the release. I suggest to resolve conversation AFTER applying the fix to not forget to do that.

Comment (kkasperczyk-no) [reply applications/matter_bridge/doc/matter_bridge_description.rst]: Sorry, it wasn't your fault :) Someone probably didn't rebase after your commit was merged and then force pushed PR, so your change was removed: https://github.com/nrfconnect/sdk-nrf/commit/b364fd802a36bd39cde9202d65735da8677bae97#diff-f250f5581599929b90c895f576af0b1ccf1b6dc333c1d6a859ac992cae861eb9R740

I will fix this.

**Pull request merged**